app/assets/javascripts/components/option-select.js

```diff
@@ -175,58 +196,65 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     } else {
       this.close()
     }
-    e.preventDefault()
+    if (e) {
```

Comment:
Interesting... why is this check necessary? I think this might be something to do with a previous comment I made, about the call to this function no longer using `bind`. The event (`e`) should always be included.

Reply:
See comment [here](https://github.com/alphagov/finder-frontend/pull/2525#discussion_r654400790)